### PR TITLE
implemented exercise acronym

### PR DESCRIPTION
**Pull request by querenker**

### CODE REVIEW

**Comment by yurrriq** [comment]
Thanks, I'll try to review this weekend.

File: exercises/acronym/src/example.clj [hunk 1]
```diff
@@ -0,0 +1,8 @@
+(ns acronym
+  (:require [clojure.string :as str]))
+
+(defn acronym [text]
+  (->> (re-seq #"[A-Z]+[a-z]*|[a-z]+" text)
+       (map first)
+       (map str/upper-case)
```
Comment (yurrriq) [inline exercises/acronym/src/example.clj]: Why not do `(->> ... (apply str) (str/upper-case))`?

File: exercises/acronym/src/example.clj [hunk 2]
```diff
@@ -0,0 +1,8 @@
+(ns acronym
+  (:require [clojure.string :as str]))
+
+(defn acronym [text]
+  (->> (re-seq #"[A-Z]+[a-z]*|[a-z]+" text)
+       (map first)
```
Comment (yurrriq) [inline exercises/acronym/src/example.clj]: Couldn't we use group capturing to obviate the need for `(->> ... (map first) ...)`?

File: exercises/acronym/src/example.clj (hunk 1 above)
Comment (querenker) [inline exercises/acronym/src/example.clj]: Thanks, that's a good point and I'll fix this

File: exercises/acronym/src/example.clj (hunk 2 above)
Comment (querenker) [inline exercises/acronym/src/example.clj]: I can't find a good solutions without impairing the readability. But maybe I miss something (I am relatively new to Clojure)

**Comment by yurrriq** [comment]
:+1: for favoring readability over micro-optimizations. I think this is good to merge then if you do.

**Comment by haus** [comment]
@querenker looks like the PR has a conflict with config.json. If you're able to resolve it, I can merge this in. If you need a hand resolving the conflict, let me know.

**Comment by querenker** [comment]
Now the merge conflict should be solved.

**Comment by haus** [comment]
Thanks @querenker !

**Pull request merged**